pkgs/development/compilers/intel-graphics-compiler/default.nix

```diff
@@ -1,6 +1,8 @@
-{ lib, stdenv
+{ lib
+, stdenv
 , fetchFromGitHub
 , cmake
+, runCommandLocal
 , pkg-config
 
```

Comment:
what's the deal with the blank line? Maybe a nixpkgs-fmt is in order?

Reply:
was there before, but is now removed